src/Promitor.Agents.Scraper/Validation/MetricDefinitions/ResourceTypes/ServiceBusNamespaceMetricValidator.cs

```diff
@@ -18,9 +18,8 @@ public IEnumerable<string> Validate(MetricDefinition metricDefinition)
 
             var errorMessages = new List<string>();
 
-            var configuredDimension = metricDefinition.AzureMetricConfiguration?.Dimension?.Name;
-            var isEntityNameDimensionConfigured = string.IsNullOrWhiteSpace(configuredDimension) == false && configuredDimension.Equals(EntityNameDimension, StringComparison.InvariantCultureIgnoreCase);
-
+            var isEntityNameDimensionConfigured = metricDefinition.AzureMetricConfiguration?.Dimensions?.Where(dimension => dimension.Name.Equals(EntityNameDimension, StringComparison.InvariantCultureIgnoreCase)).Any() ?? false;
```

Comment:
See comment in src/Promitor.Agents.Scraper/Validation/MetricDefinitions/ResourceTypes/EventHubsMetricValidator.cs

Reply:
Done